.circleci/config.yml

```diff
@@ -375,7 +375,7 @@ jobs:
           command: |
             source activate ~/project/test-environment-27
             conda install --channel conda-forge sphinx future
-            pip install sphinxcontrib-bibtex
+            pip install sphinxcontrib-bibtex <=0.4.2
```

Comment:
I needed quotes around that in bash, pip install "sphinxcontrib-bibtex<=0.4.2"

Reply:
Quotes were already added in a later commit, but thanks